src/zmqsubscriber/zmqsubscriber_test.go

```diff
@@ -52,32 +52,54 @@ func getTestZMQSubscriber(t *testing.T, env test.TestEnv) *ZMQSubscriber {
 func TestZMQSubscriber(t *testing.T) {
 	const waitTimeout = 5 * time.Second
 
-	env := test.NewTestEnv()
+	env, err := test.NewTestEnv()
+	if err != nil {
+		t.Errorf("Could not create a new test environment: %s", err)
+	}
 	defer env.Quit()
 
+	addressSendFrom, err := env.GetNewAddress()
+	if err != nil {
+		t.Errorf("could not generate a new bitcoin address: %s", err)
+	}
+
+	// Generate 101 blocks to have spendable UTXOs.
+	env.GenerateToAddress(101, addressSendFrom)
```

Comment:
I'm ok with removing `panic()` for recoverable errors, but we have to do the err checks then